geoips/interfaces/base.py

```diff
@@ -388,66 +387,14 @@ def _plugin_yaml_to_obj(cls, name, yaml_plugin, obj_attrs={}):
         plugin_type = f"{plugin_interface_name}Plugin"
 
         plugin_base_class = BaseYamlPlugin
+        # from IPython import embed as shell
+
+        # shell()
         if hasattr(cls, "plugin_class") and cls.plugin_class:
             plugin_base_class = cls.plugin_class
 
         return type(plugin_type, (plugin_base_class,), obj_attrs)(yaml_plugin)
 
-    def _create_unvalidated_plugins_cache(self, yaml_plugins):
-        """Create a cache of unvalidated plugin yamls.
-
-        These will be validated when they are actually used.
-        """
-        cache = {}
-        # If this is a list, split out all of the subs and store them all
```

Comment:
@evrose54 @jsolbrig Note I removed these now-unused methods.  Good riddance.

Reply:
See: https://github.com/NRLMMD-GEOIPS/geoips/pull/380/files

Reply:
This has been merged. Resolving.